File: Modul 05/01/PRAK501-2410817310020-M. Rizki Dinar.py
def max_bilangan(a, b, c, d):
    arr = [a, b, c, d]
    max_now = arr[0]
    for num in arr:
        if num > max_now:
            max_now = num
    return max_now

a, b, c, d = map(int, input().split())
hasil = max_bilangan(a, b, c, d)
print(hasil)

File: Modul 05/01/PRAK501-2410817310020-M. Rizki Dinar.c
#include <stdio.h>

int MaxBilangan(int a, int b, int c, int d) {
    int arr[] = {a,b,c,d};
    int maxNow = arr[0];
    for (int i = 0; i < 4; i++) {
        if (arr[i] > maxNow) {
            maxNow = arr[i];
        }
    }
    return maxNow;
}

int main() {
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan(a, b, c, d);
    printf("%d", hasil);
    return 0;
}
